examples/src/main/kotlin/ai/koog/agents/example/client/BedrockAgent.kt

```diff
@@ -0,0 +1,48 @@
+package ai.koog.agents.example.client
+
+import ai.koog.agents.core.agent.AIAgent
+import ai.koog.agents.core.tools.ToolRegistry
+import ai.koog.agents.core.tools.reflect.asTools
+import ai.koog.agents.example.ApiKeyService
+import ai.koog.agents.example.simpleapi.Switch
+import ai.koog.agents.example.simpleapi.SwitchTools
+import ai.koog.prompt.executor.clients.bedrock.BedrockClientSettings
+import ai.koog.prompt.executor.clients.bedrock.BedrockModels
+import ai.koog.prompt.executor.llms.all.simpleBedrockExecutor
+import kotlinx.coroutines.runBlocking
+
+fun main() = runBlocking {
```

Comment:
nit: Please specify return type (`Unit`) explicitly, otherwise there's a warning about potential non-Unit return type in main (since `runBlocking` can return some value).